tests/framework-tests/core/html/Keyboard-a11y-test.html

```diff
@@ -14,9 +14,17 @@
 
     <!--  These are the jqUnit test js files -->
     <script type="text/javascript" src="../../../lib/qunit/js/qunit.js"></script>
+    <script src="/testem.js"></script>
     <script type="text/javascript" src="../../../../src/framework/core/js/Fluid.js"></script>
     <script type="text/javascript" src="../../../test-core/jqUnit/js/jqUnit.js"></script>
 
+
+    <!-- These are required not for the tests themselves, but for the coverage client -->
+    <script type="text/javascript" src="../../../../src/framework/core/js/DataBinding.js"></script>
+    <script type="text/javascript" src="../../../../src/framework/core/js/FluidIoC.js"></script>
```

Comment:
Could you double check if these 2 files are needed? The same comment applies to other spots.